content/en/security/application_security/setup/python/aws_fargate.md

```diff
@@ -0,0 +1,167 @@
+---
+code_lang: aws-fargate
+type: multi-code-lang
+code_lang_weight: 60
+title: Set up App and API Protection for Python on AWS Fargate
+further_reading:
+- link: "/security/application_security/how-it-works/"
+  tag: "Documentation"
+  text: "How App and API Protection Works"
+- link: "/security/default_rules/?category=cat-application-security"
+  tag: "Documentation"
+  text: "OOTB App and API Protection Rules"
+- link: "/security/application_security/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshooting App and API Protection"
+---
+
+{{% app_and_api_protection_python_overview %}}
+
+## Prerequisites
+
+- AWS Fargate environment
+- Python application containerized with Docker
+- AWS CLI configured with appropriate permissions
+- Your Datadog API key
+- Datadog Python tracing library (see version requirements [here][1])
+
+## 1. Installing the Datadog Agent
+
+Install the Datadog Agent in your Fargate task definition:
+
+```json
+{
+  "containerDefinitions": [
+    {
+      "name": "datadog-agent",
+      "image": "public.ecr.aws/datadog/agent:latest",
+      "environment": [
+        {
+          "name": "DD_API_KEY",
+          "value": "<YOUR_API_KEY>"
+        },
+        {
+          "name": "DD_APM_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "DD_APM_NON_LOCAL_TRAFFIC",
+          "value": "true"
+        }
+      ]
+    }
+  ]
+}
+```
+
+## 2. Enabling App and API Protection monitoring
+
+{{% app_and_api_protection_python_navigation_menu %}}
+
+{{% appsec-remote-config-activation %}}
+
+### Manually enabling App and API Protection monitoring
+Install the Datadog Python tracing library in your application environment:
+
+```dockerfile
+RUN pip install ddtrace
+```
+
```

Comment:
You missing a transition sentence here between pip install and the next dropdowns.

You should add a sentence like:
```suggestion

Configure and run your service with Datadog:
```

Reply:
:ok_hand: